packages/web-components/src/globals/mixins/stable-selector.ts

```diff
@@ -44,6 +44,13 @@ const StableSelectorMixin = <T extends Constructor<HTMLElement>>(Base: T) => {
         .constructor as typeof StableSelectorMixinImpl;
       if (stableSelector) {
         this.dataset.autoid = stableSelector;
+
+        var css = `${this.tagName.toLowerCase()}:not(:defined) { display: none; }`,
+          head = document.head || document.getElementsByTagName('head')[0],
+          style = document.createElement('style');
+
+        head.appendChild(style);
```

Comment:
Should it check if `<style>` already exists, and if so just append to that? Otherwise there could be lots of lots of style tags in `<head>`

Reply:
updated!